docs/source/examples/capacity_expansion_tutorial.ipynb

```diff
@@ -0,0 +1,344 @@
+      "n_gen  |  n_eval  | n_nds  |      eps      |   indicator  \n",
+      "==========================================================\n",
+      "     1 |        5 |      5 |             - |             -\n",
+      "     2 |       10 |      5 |  0.000000E+00 |             f\n",
+      "     3 |       15 |      5 |  0.000000E+00 |             f\n",
+      "     4 |       20 |      5 |  0.000000E+00 |             f\n",
+      "     5 |       25 |      5 |  0.000000E+00 |             f\n",
+      "     6 |       30 |      5 |  0.000000E+00 |             f\n",
+      "     7 |       35 |      5 |  0.000000E+00 |             f\n",
+      "     8 |       40 |      5 |  0.000000E+00 |             f\n",
+      "     9 |       45 |      5 |  0.000000E+00 |             f\n",
+      "    10 |       50 |      5 |  0.000000E+00 |             f\n",
+      "    11 |       55 |      5 |  0.000000E+00 |             f\n",
+      "    12 |       60 |      5 |  0.000000E+00 |             f\n",
+      "    13 |       65 |      5 |  0.000000E+00 |             f\n",
+      "    14 |       70 |      5 |  0.000000E+00 |             f\n",
+      "    15 |       75 |      5 |  0.000000E+00 |             f\n",
+      "    16 |       80 |      5 |  0.000000E+00 |             f\n",
+      "    17 |       85 |      5 |  0.000000E+00 |             f\n",
+      "    18 |       90 |      5 |  0.000000E+00 |             f\n",
+      "    19 |       95 |      5 |  0.000000E+00 |             f\n",
+      "    20 |      100 |      5 |  0.000000E+00 |             f\n"
+     ]
+    }
+   ],
+   "source": [
+    "algorithm = NSGA2(pop_size=5)\n",
+    "\n",
+    "res = minimize(problem,\n",
+    "               algorithm,\n",
+    "               termination=('n_gen', 20),\n",
+    "               seed=1,\n",
+    "               verbose=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 51,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([[0.07145398, 0.14901718],\n",
+       "       [0.11407881, 0.19480258],\n",
+       "       [0.00590286, 0.23405426],\n",
+       "       [0.1148452 , 0.19480258],\n",
+       "       [0.00106922, 0.25836035]])"
+      ]
+     },
+     "execution_count": 51,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "res.X"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "base",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.12"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
This seems unfinished.  Isn't there a final plot of the wind capacity expansion that should be created, or something similar?  What do you do with/after setting up the Pymoo algorithm?